students/TinaB/lesson07/test_html_render.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+
+import html_render as hr
+from io import StringIO
+import unittest
+
+
+class Test_Html_Render(unittest.TestCase):
+
+    def setUp(self):
+        self.html = hr.Html()
+        self.head = hr.Head(hr.Title('The Title is Title'))
+        self.body = hr.Body()
+        self.paragraph = hr.P('This is my paragraph', style="color:blue;")
+        self.horizontal_rule = hr.Hr()
+        self.anchor = hr.A('http://chicktech.org', 'link')
+        self.unorderedlist = hr.Ul(id="UL_List", style="line-height:100%")
+        self.headertwo = hr.H(3, 'This is a Header 3 ')
+        self.meta = hr.Meta(charset="UTF-8")
+
+    def test_append(self):
+        """Test the append method"""
+        self.html.append(
+            'This is a test of the append method...')
+        self.assertTrue(self.html.content == ['This is a test of the append method...'],
+                        msg=self.html.content)
+
+    def test_element_content(self):
+        """testing append method through element reference"""
+        test_element = hr.Element("this is some content")
+        self.assertEqual(test_element.content, ["this is some content"])
+        test_element.append("Adding additional content")
+        self.assertEqual(test_element.content, [
+                         "this is some content", "Adding additional content"])
+
+    def test_element_tag(self):
+        """Testing override tag properties"""
+        self.assertEqual(self.body.tag, 'body')
+        self.assertEqual(self.head.tag, 'head')
+        self.assertEqual(self.paragraph.tag, 'p')
+        self.assertEqual(self.horizontal_rule.tag, 'hr')
+        self.assertEqual(self.anchor.tag, 'a')
```

Comment:
it would be better to test each tag as far as rendering vs just testing the attribute.